File: swedisch-vocab-app/src/main/java/de/isah/vocabtrainer/InfoActivity.java
package de.isah.vocabtrainer;

import android.os.Bundle;
import android.widget.TextView;

import de.isah.vocabtrainer.logging.SwedishVocabAppLogger;


/**
 *
 */
public class InfoActivity extends VocabTrainerAppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        SwedishVocabAppLogger.log("on create", InfoActivity.class);
        setContentView(R.layout.activity_info);

        TextView textView1 = findViewById(R.id.textViewAppInfoName);
        textView1.setText(R.string.app_name);

        TextView textView2 = findViewById(R.id.textViewAppInfoVersion);
        textView2.setText(R.string.app_version);
    }

}

File: swedisch-vocab-app/src/main/kotlin/de/isah/vocabtrainer/InfoActivity.kt
package de.isah.vocabtrainer

import android.os.Bundle
import android.widget.TextView
import de.isah.vocabtrainer.logging.SwedishVocabAppLogger

class InfoActivity : VocabTrainerAppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_info)
        SwedishVocabAppLogger.log("on create", InfoActivity::class.java)

        val textView1 = findViewById<TextView>(R.id.textViewAppInfoName)
        textView1.text = getString(R.string.app_name)

        val textView2 = findViewById<TextView>(R.id.textViewAppInfoVersion)
        textView2.text = getString(R.string.app_version)
    }
}